src/main/java/seedu/exchangecoursemapper/parser/Parser.java

```diff
@@ -78,7 +78,7 @@ public void processUserInput(String userInput, Storage storage) {
         } else if (command.equals(OBTAIN)) {
             new ObtainContactsCommand().execute(input);
         } else if (input.equals(LIST_MAPPED)) {
-            new ListPersonalTrackerCommand(storage).execute(input);
+            new ListPersonalTrackerCommand().execute(input, storage);
```

Comment:
The original `new ListPersonalTrackerCommand(storage).execute(input);` will work better here.

Reply:
Good catch! Thank you for helping out with the merge conflict, will edit it accordingly! 👍